package.json

```diff
@@ -12,7 +12,7 @@
   "scripts": {
     "start": "rescripts start",
     "build": "rescripts build",
-    "build:package": "tsc --noEmit false --outDir package --jsx react --declaration true --allowJs false --isolatedModules false --target es5 --module commonjs && mv package/exports.d.ts package/index.d.ts && mv package/exports.js package/index.js",
+    "build:package": "tsc --noEmit false --outDir package --jsx react --declaration true --allowJs false --isolatedModules false --target es5 --module commonjs && mv package/exports.d.ts package/index.d.ts && mv package/exports.js package/index.js && cp src/components/cardListItem/cardListItem.css package/components/cardListItem && cp src/components/logsReact/logsReact.css package/components/logsReact && cp src/components/methodList/methodList.css package/components/methodList && cp src/components/methodListItem/methodListItem.css package/components/methodListItem",
```

Comment:
Can this use wildcards to trim up a little? Maybe something like `find src/components -maxdepth 1 -type d | xargs cp {}/*.css package/components/{}/` (that's just off the cuff, you'll need to check me).
This way if components are added you won't have to muck about remembering to add to the command.

Reply:
Minuto. This is def borken. See if I can whip something that actually works up quick.
Or is there any pre-built way (eg. script/npm task) of shifting css files around like this?

Reply:
```
find src/components/* -maxdepth 1 -type d | while read d; do cp "$d"/*.css "$(echo $d | sed 's/src/package/')/"; done || true
```

Reply:
Here's a patch _hif you want_ to use it. I don't have push rights, otherwise I'd just push and let you revert or tweak.

```
commit 966a5735ab378b7b554b7634e7fb51131830565e (HEAD -> fix/package-css)
Author: meows <b5c6@protonmail.com>
Date:   Tue Jun 16 14:04:43 2020 -0500

    package.json: feat(build:package script): cp any css files from src to package dirs, respectively
    
    This is just a miniature script to copy necessary
    css files from src to package dirs.
    
    Its nice because it will handle any more
    components that might get added.
    
    Signed-off-by: meows <b5c6@protonmail.com>

diff --git a/package.json b/package.json
index 5f04e9e..e2d0c8a 100644
--- a/package.json
+++ b/package.json
@@ -12,7 +12,7 @@
   "scripts": {
     "start": "rescripts start",
     "build": "rescripts build",
-    "build:package": "tsc --noEmit false --outDir package --jsx react --declaration true --allowJs false --isolatedModules false --target es5 --module commonjs && mv package/exports.d.ts package/index.d.ts && mv package/exports.js package/index.js && cp src/components/cardListItem/cardListItem.css package/components/cardListItem && cp src/components/logsReact/logsReact.css package/components/logsReact && cp src/components/methodList/methodList.css package/components/methodList && cp src/components/methodListItem/methodListItem.css package/components/methodListItem",
+    "build:package": "tsc --noEmit false --outDir package --jsx react --declaration true --allowJs false --isolatedModules false --target es5 --module commonjs && mv package/exports.d.ts package/index.d.ts && mv package/exports.js package/index.js && find src/components/* -maxdepth 1 -type d | while read d; do cp \"$d\"/*.css \"$(echo $d | sed 's/src/package/')/\"; done && echo 'You can safely ignore _cp_ errors, if any.'",
     "lint": "tslint --fix -p .",
     "test": "npm run lint && CI=true rescripts test --coverage --passWithNoTests"
   },
```

Reply:
I wish I refreshed this page earlier. Just pushed a fix, let me know which one you think is best

Reply:
All good, either way!